test/schema/fixtures/CodedDescendent.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "$schema": "http://standardhealthrecord.org/schemas/shr/extended-schema.schema.json",
+    "id": "http://standardhealthrecord.org/schemas/shr/test/CodedDescendent.schema.json",
+    "description": "It is a coded element descending from foobar",
+    "$ref": "http://standardhealthrecord.org/schemas/shr/types.schema.json#/definitions/code"
+  }
```

Comment:
This fixture doesn't contain the `descendingFrom` property.  Since you didn't actually implement `descendingFrom` in the exporter yet, it passes -- but it's a little deceiving that we have an incorrect fixture so this test passes even though the capability isn't implemented.